bin/viz_pipe

```diff
@@ -0,0 +1,22 @@
+dotFile=$1.dot
+
+echo "digraph G {\n	node [style=bold, shape=box];" > $dotFile
+
+valid_plugins $1 | while read line; do
+  # The next line will edit a valid plugin line into the form of "a"->"b"->"c"
+  # We first start by using awk to get the necessary fields. The default field delimeter is a space.
+  # We then print the necessary fields that describe an edge in the directed graph. 
+  # $2 is the Plugin name, $4 is the input file name, and $6 is the output file name.
+  # The OFS flag lets us set the output delimeter to ->. 
+  # At this point of the pipeline we have a->b->c
+  # We then need to use sed to wrap the vertices.
+  # We create a capturing group by using the regex expression: ([^->]+),
+  # this will capture any character into a group except - and >
+  # We then want to wrap the captured groups with quotes we do this with "\1"
+  
+  echo "\t`echo "$line" | awk 'OFS="->" {print $4,$2,$6}' | sed -E 's/([^->]+)/"\1"/g'`" >> $dotFile
```

Comment:
There's an issue with the tab escape character. 

This is the contents of the dot file generated by the script:

 digraph G {\n   node [style=bold, shape=box];
\t"abund.csv"->"CSVNormalize"->"abund.norm.csv"
\t"abund.norm.csv"->"Spearman"->"spearman.csv"
\t"spearman.csv"->"CSVPad"->"network.csv"
\t"network.csv"->"GPUATria"->"ATria.noa"
\t"network.csv"->"CSV2GML"->"network.gml"
}

Reply:
Interesting... Does this still happen if you use `printf `instead of `echo`?

Reply:
I updated `echo `with `printf`. Could you try now?

Reply:
That fixed it.